doc/sphinxext/gen_commands.py

```diff
@@ -51,8 +51,8 @@ def generate_commands_rst(app=None):
         from sphinx.util.display import status_iterator
     except Exception:
         from sphinx.util import status_iterator
-    root = Path(__file__).parent.parent.parent.absolute()
```

Comment:
Even on this branch:
```
$ git grep "\.parent\.parent" | wc -l
70
```
but they are now at least all isolated to `tests` files, I made the others more compact in this PR